package/tests/test_models/test_graph/test_graph_nodes.py

```diff
@@ -147,6 +150,29 @@ def test_create_data_node(
         assert data_node.modular_pipelines == expected_modular_pipelines
         assert not data_node.is_plot_node()
 
+    @pytest.mark.parametrize(
+        "dataset_name,original_name,pretty_name",
+        [
+            ("dataset@spark", "dataset", "Dataset"),
```

Comment:
You don't need to parametrize if it's only one instance. But it's fine this way.